tutorials/holohub_operators_external_applications/README.md

```diff
@@ -439,12 +439,87 @@ cmake -DCMAKE_VERBOSE_MAKEFILE=ON ..
 
 See the `main.cpp`, `main.py`, and `CMakeLists.txt` files in this directory for complete working examples that demonstrate how to use the AJA source operator from Holohub in both C++ and Python applications.
 
+## Using Holohub CLI in External Projects
+
+The Holohub CLI provides convenient command-line tools for managing Holohub applications, including building, running, and testing operators and applications. You can easily integrate the CLI into your external projects to leverage these functionalities.
+
+### Adding holohubCLI_wrapper to Your Project
+
+To use the Holohub CLI in your external project, simply copy the `holohubCLI_wrapper` script from this tutorial to the root of your project.:
+
+```bash
+# Copy the CLI script to your project root
+cp tutorials/holohub_operators_external_applications/holohubCLI_wrapper /path/to/your/project/myprojectCLI
+
+# Make it executable
+chmod +x /path/to/your/project/myprojectCLI
+```
+
+Edit the line in `myprojectCLI` to match the name of the script:
+
+```bash
+export HOLOHUB_CMD_NAME='./holohubCLI_wrapper'
+```
+
+### Project Structure with CLI
+
+Your project structure will look like this:
```

Comment:
Should we suggest a `.gitignore` as well to avoid the HoloHub sparse checkout from being checked in in downstream project repos?